integrationtests/src/test/java/org/neo4j/procedure/ProcedureIT.java

```diff
@@ -1129,6 +1129,20 @@ public void shouldUseGuardToDetectTransactionTermination() throws Throwable
         db.execute( "CALL org.neo4j.procedure.guardMe" );
     }
 
+    @Test
+    public void shouldMakeTransactionToFail() throws Throwable
+    {
+        //When
+        try ( Transaction ignore = db.beginTx() )
+        {
+            db.createNode( Label.label( "Person" ) );
+        }
+        Result result = db.execute( "CALL org.neo4j.procedure.failure" );
```

Comment:
I think calling the procedure `failingPersonCount` would make this test much easier to read. Took a little time to realize why a node was needed!